--- libarchive-c-2.9/tests/test_rwx.py.orig	2019-10-20 14:34:28.000000000 +0200
+++ libarchive-c-2.9/tests/test_rwx.py	2024-05-19 21:48:46.268723039 +0200
@@ -9,7 +9,10 @@ import libarchive
 from libarchive.entry import format_time
 from libarchive.extract import EXTRACT_OWNER, EXTRACT_PERM, EXTRACT_TIME
 from libarchive.write import memory_writer
-from mock import patch
+try:
+    from unittest.mock import patch
+except ImportError:
+    from mock import patch
 import pytest
 
 from . import check_archive, in_dir, treestat
